src/strategies/decentraland-wearable-rarity/index.ts

```diff
@@ -0,0 +1,83 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+export const author = '2fd';
+export const version = '0.1.0';
+
+const DECENTRALAND_COLLECTIONS_SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/decentraland/collections-ethereum-mainnet',
+  '3': 'https://api.thegraph.com/subgraphs/name/decentraland/collections-ethereum-ropsten',
+  '137': 'https://api.thegraph.com/subgraphs/name/decentraland/collections-matic-mainnet',
+  '80001': 'https://api.thegraph.com/subgraphs/name/decentraland/collections-matic-mumbai',
+};
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+
+  // initialize scores
+  const scores = {};
+  for (const address of addresses) {
+    scores[getAddress(address)] = 0
+  }
+
+  // if graph doesn't exists return automaticaly
+  if (!DECENTRALAND_COLLECTIONS_SUBGRAPH_URL[network]) {
+    return scores
+  }
+
+  // initialize multiplers and params
+  const multiplers = options.multipliers || {  };
+  const params = {
+    nfts: {
+      __args: {
+        where: {
+          itemType_in: ['wearable_v1',  'wearable_v2', 'smart_wearable_v1'],
```

Comment:
We must filter by contract address too. We are not going to compute all the L1 wearables, just a few of them